### DISCUSSION TITLE
posts/Go/golang-unit-test-4/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#320] [Announcements]
https://www.liwenzhou.com/posts/Go/golang-unit-test-4/ 

 李文周的Blog Go语言测试 单元测试怎么写,go test 执行测试,接口mock工具,GoStub是常用的打桩工具,为全局变量打桩。gomock是官方出品的mock工具，支持mock接口类型,也支持为方法打桩,assert,断言,如何进行接口mock,mock接口教程,go mock怎么用,gostub怎么用,如何编写单元测试

**Comment by Albelt** [comment]
辛苦了，中秋节还在更新文章

**Comment by Anaethesia** [comment]
通俗易懂，看完直接上手写:)
